packages/datatrak-web/src/features/Survey/Components/SurveyToolbar.tsx

```diff
@@ -42,18 +43,25 @@ const SurveyTitle = styled(Typography).attrs({
   align-items: center;
 `;
 
+const CountryName = styled.span`
+  padding-left: 0.3rem;
+  font-weight: ${({ theme }) => theme.typography.fontWeightRegular};
+`;
+
 export const SurveyToolbar = () => {
   const { surveyCode } = useParams();
   const { screenNumber, numberOfScreens } = useSurveyForm();
   const { data: survey } = useSurvey(surveyCode);
+  const { selectedCountry } = useUserCountries();
```

Comment:
```suggestion
  const { data: { country } } = useUser();
```
Can we use the useUser query instead of the useUserCountries util. There is a lot of complexity in useUserCountries and it would be good not to tie it here unnecessarily.